Assets/Scripts/Utils/ElementInteraction/Draggable.cs

```diff
@@ -11,41 +12,64 @@ namespace Utils.ElementInteraction
     [RequireComponent(typeof(Image))]
     public class Draggable : MonoBehaviour, IDragHandler, IEndDragHandler, IBeginDragHandler
     {
+        [Tooltip("Used for showing this on top of other UI-Elements")] [SerializeField]
+        private Canvas canvas;
+
+        [Tooltip("Used for finding UI elements with rays")] [SerializeField]
+        private GraphicRaycaster graphicRaycaster;
+
         [Tooltip("The highest parent element that will be moved")] [SerializeField]
         private Transform parent;
 
+        public GraphicRaycaster GraphicRaycaster { get; private set; }
```

Comment:
You are currently using two different graphic raycasters: One which is stored in this property and fetched using GetComponentsInParent on Awake, and one which is stored in a serialised field. I guess that's not intended

Reply:
Hmm why do you use GetComponentsInParent anyway if you have a serialized field?

Reply:
Renamed it